graph/vertex/impl/AttributeVertexImpl.java

```diff
@@ -588,6 +588,16 @@ public ValueSortable<VALUE> toValueSortable() {
             return this;
         }
 
+        @Override
+        public ThingAdjacency.In ins() {
```

Comment:
This is a bugfix where we tried to traverse a Sortable wrapper of a vertex but failed to find vertices because the inner vertex contained edges, but the wrapper did not - now we use it as a pass through instead.